x-pack/legacy/plugins/code/server/lsp/ts_launcher.ts

```diff
@@ -61,7 +61,7 @@ export class TypescriptServerLauncher extends AbstractLauncher {
       log.stdout(data.toString());
     });
     p.stderr.on('data', data => {
-      log.stderr(data.toString());
+      log.stdout(data.toString());
```

Comment:
so we are always assuming the errors thrown from the lang server can be safely ignored?